hedera-node/src/test/java/com/hedera/services/store/models/UniqueTokenTest.java

```diff
@@ -35,6 +35,13 @@ void testSubject(){
 		subj.setSerialNumber(2);
 		assertEquals(2, subj.getSerialNumber());
 		assertEquals(3, subj.getMetadata().length);
+
+		subj.setMetadata(new byte[]{1, 2, 3});
+		assertEquals(3, subj.getMetadata().length);
```

Comment:
I am not sure that this test actually tests the `setMetadata` method. The perfect case would be to asser that the array of bytes are the same not asserting the length of the array